src/main/java/org/terasology/dynamicCities/settlements/SettlementEntityManager.java

```diff
@@ -132,7 +132,8 @@
 
     private int minDistance = 500;
     private int settlementMaxRadius = 150;
-    private int cyclesLeft = 2; // 1 cycle = approx. 20 seconds
+    private int cyclesLeft; // 1 cycle = approx. 20 seconds
+    private int NumberOfCyclesOnReset = 2;
```

Comment:
Pleas use lower case names for variables. If this should be a constant, it should be
```java
private static final int NUMBER_OF_CYCLES_ON_RESET = 2;
```

Reply:
```suggestion
    private int numberOfCyclesOnReset = 2;
```